### PR TITLE
Change sugar source

### PR DESCRIPTION
**Pull request by ErinBecker**
updated glucose to galactose

### CODE REVIEW

**Comment by ErinBecker** [comment]
@karenword can you please merge?

**Comment by karenword** [comment]
No!! It’s wrong.

File: _episodes/01-introduction.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ impossible for more complex eukaryotic systems. In [Blount et al 2012](Lenski_pa
 minimal medium. This medium was supplemented with citrate which *E. coli* cannot metabolize in the aerobic conditions of the experiment. Sequencing of the populations at regular time points reveals
 that spontaneous citrate-using mutants (Cit+) appeared in a population of *E.coli* (designated Ara-3) at around 31,000 generations. It should be noted that spontaneous Cit+ mutants are extraordinarily
 rare - inability to metabolize citrate is one of the defining characters of the *E. coli* species. Eventually, Cit+ mutants became the dominant population as the experimental growth medium contained a
-high concentration of citrate relative to glucose.  
+high concentration of citrate relative to galactose.
```
Comment (lexnederbragt) [inline _episodes/01-introduction.md]: I think this is wrong...

**Comment by lmavila** [comment]
no!

**Pull request closed without merging**